### PR TITLE
[Feature] Optimize router & metadata Menu

### PR DESCRIPTION
**Pull request by zhangmo8**
### Purpose

Optimize router & metadata Mena.

### CODE REVIEW

File: paimon-web-ui-new/src/router/modules/metadata.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. */
+
+export default {
+  path: '/metadata',
+  name: 'metadata',
+  meta: { title: '元数据查询' },
+  component: () => import('@/views/metadata')
+}
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/router/modules/metadata.ts:23]: New line at the end line of file.

File: paimon-web-ui-new/src/views/metadata/components/menu-tree/index.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+              ]
+            }
+          ]
+        }
+      ],
+      filterValue: '',
+      selectedKeys: []
+    })
+
+    // [
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/views/metadata/components/menu-tree/index.tsx:76]: Remove the comments.

**Review by SteNicholas — COMMENTED** [review]
@zhangmo8, thanks for the contribution. I left minor comments for this pull request. PTAL.

File: paimon-web-ui-new/src/router/modules/metadata.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License. */
+
+export default {
+  path: '/metadata',
+  name: 'metadata',
+  meta: { title: '元数据查询' },
```
Comment (SteNicholas) [inline paimon-web-ui-new/src/router/modules/metadata.ts:21]: Could all titles use English?

**Review by labbomb — APPROVED** [review]
LGTM.

**Review by SteNicholas — APPROVED** [review]
LGTM. Thanks for @zhangmo8 updates.

**Pull request merged**